etc/init/spinnaker-monitoring.conf

```diff
@@ -0,0 +1,3 @@
+start on filesystem or runlevel [2345]
+
```

Comment:
We do not have it protected, its readable for all users on the system. This is not a blocker for us.